source/markdown/data_controle_2024.Rmd

```diff
@@ -0,0 +1,1572 @@
+---
+title: "Controle data en data preparatie pipeline"
+author: "Ward Langeraert"
+date: "`r Sys.Date()`"
+output:
+  bookdown::html_document2:
+    code_folding: hide
+    toc: true
+    toc_float: true
+    toc_collapsed: true
+knit: (function(inputFile, encoding) {
+  rmarkdown::render(inputFile, encoding = encoding, output_dir = "../../output/rapporten/markdown/2024") })
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include=FALSE}
+# set up
+library(knitr)
+opts_chunk$set(
+  echo = TRUE,
+  warning = FALSE,
+  message = FALSE,
+  out.width = "100%"
+)
+opts_knit$set(root.dir = here::here())
+```
+
+```{r}
+# packages
+library(tidyverse)
+library(sf)
+library(mapview)
+library(leaflet)
+library(targets)
+
+# Conflicts
+conflicted::conflicts_prefer(dplyr::filter)
+conflicted::conflicts_prefer(dplyr::select)
+
+# Source
+source(here::here("source", "R", "tar_read_sf.R"))
+
+# Paths
+mbag_dir <- here::here()
+targets_store <- here::here("source", "targets", "data_preparation", "_targets")
+```
+
+# Doel
+
+Data preparatie verloopt via een targets pipeline waarbij verschillende stappen elkaar opvolgen.
+
+```{r, message=FALSE}
+tar_visnetwork(script = paste0(targets_store, ".R"),
+               store = targets_store)
+```
+
+In deze markdown controleren we ...
+
+1. of deze pipeline de verwachte stappen correct doorloopt
+2. of in de finale data, op het einde van de pipeline, onverwachte zaken, uitschieters ... aanwezig zijn
+3. of de data klaar zijn voor publicatie op GBIF
+
+We doen dit in dit rapport voor de data van 2018-2024.
+
+# Controle data preparatie pipeline {#controle-pipeline}
+## Amersfoord naar Lambert coördinaten (crs_pipeline)
+
+In de eerste stap worden Amersfoord coördinaten naar Lambert coördinaten omgezet.
+Dit gebeurt via een functie die op verschillende datasets goed getest is.
+
+## Selectie punten MBAG steekproefkader (select_sampled_points) {#steekproefkader}
+
+In de tweede stap selecteren we de punten uit de totale dataset die tot het MBAG steekproefkader behoren.
+Dit gebeurt via een "inner join" wat wil zeggen dat we kunnen nagaan of telpunten uit het steekproefkader niet bezocht zijn (bewust of onbewust).
+
```

Comment:
I think you mean "steekproef" instead of "steekproefkader" in this section?

Reply:
fixed in this commit: [abee402](https://github.com/inbo/mbag-mas/pull/37/commits/abee4024effb3d9f8051d6d1a4bea52cccc72de0)